tests/e2e/e2e_common.go

```diff
@@ -110,7 +110,7 @@ const (
 	invalidFSType                             = "ext10"
 	k8sPodTerminationTimeOut                  = 7 * time.Minute
 	k8sPodTerminationTimeOutLong              = 10 * time.Minute
-	k8sVmPasswd                               = "ca$hc0w"
+	k8sVmPasswd                               = "C$!Fvt#8"
```

Comment:
Ideally, this should be read from the pipeline env variable.
password should not be checked-in in the repository.